datahub-web-react/src/app/ingest/source/builder/RecipeForm/azure.ts

```diff
@@ -0,0 +1,174 @@
+import { RecipeField, FieldType, setListValuesOnRecipe } from './common';
+
+const validateURL = (fieldName) => {
```

Comment:
it looks like this function was copied from `csv.ts` - let's bring that function to a new shared `utils.ts` file and import it in the three recipes instead of copying it twice.

Then, it would be nice to have unit tests on that utility function

Reply:
updated!